datasource-local/src/main/java/cmm/apps/esmorga/datasource_local/event/EventLocalDatasourceImpl.kt

```diff
@@ -26,4 +26,26 @@ class EventLocalDatasourceImpl(private val eventDao: EventDao) : EventDatasource
     override suspend fun deleteCacheEvents() {
         eventDao.deleteAll()
     }
+
+    override suspend fun joinEvent(eventId: String) {
+        val events = eventDao.getEvents().map {
+            if (it.localId == eventId) {
+                it.copy(localUserJoined = true)
+            } else {
+                it
+            }
+        }
+        eventDao.insertEvent(events)
```

Comment:
insertEvent singular accepts a list of events?

Reply:
yes, ` suspend fun insertEvent(event: List<EventLocalModel>)` ... I can rename it to insertEvents